### PR TITLE
Angular 13 support + some additions

### PR DESCRIPTION
**Pull request by KevinBonnoron**
Hi !

This PR :
- add support for angular 13,
- allow user to change the trigger to show the mat-popover (toggle / hover) with default to hover,
- convert deprecated protractor configurations to use cypress,

### CODE REVIEW

**Comment by maxisam** [comment]
Thank you for making this PR, I will review it later today.

**Review by maxisam — COMMENTED** [review]
Do you think we should keep cypress/videos/app_spec.ts.mp4  in the repo? I just wanna make sure this is not add by accident. I haven't used cypress yet but I do wanna try it sometime.

File: install.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const { execSync } = require('child_process');
```
Comment (maxisam) [inline install.js]: do we still need this file?

File: projects/ngx-mat-popover/package.json [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-  "version": "0.0.1",
-  "author": {
-    "name": "Sam Lin",
-    "email": "maxisam@gmail.com"
-  },
+  "version": "0.0.2",
   "homepage": "https://github.com/maxisam/ngx-mat-popover",
   "license": "MIT",
-  "publishConfig": {},
+  "publishConfig": {
```
Comment (maxisam) [inline projects/ngx-mat-popover/package.json]: should we change this?

**Review by maxisam — COMMENTED** [review]
Overall I think it look great. Just wanna double check some details

File: install.js (hunk 1 above)
Comment (KevinBonnoron) [reply install.js]: No, i will remove it

File: projects/ngx-mat-popover/package.json (hunk 2 above)
Comment (KevinBonnoron) [reply projects/ngx-mat-popover/package.json]: No sorry it's a mistake when i was trying the lib in another project. Reverting to original

**Comment by KevinBonnoron** [comment]
The cypress/video is a mistake, i'm going to add it to gitignore and remove it. Thanks for review

**Review by maxisam — APPROVED** [review]

**Comment by maxisam** [comment]
Could you check 1.0.0-beta1 thx

**Comment by KevinBonnoron** [comment]
Seems fine to me. Thanks !

**Pull request merged**